src/catalog/inventory-product-alerts.md

```diff
@@ -36,12 +36,17 @@ _List of Product and Price Alert Subscriptions_
     - To offer alerts when out-of-stock products become available again, set **Allow Alert When Product Comes Back in Stock** to `Yes`.
 
       {:.bs-callout-info}
-      The _Sign up to get notified when this product is back in stock_ message appears only when **Display Out of Stock Products** is set to `Yes` (located in the Configuration at Catalog > Inventory > Stock Options).
+      The _Notify me when this product is in stock_ message appears only when **Display Out of Stock Products** is set to `Yes` (located in the Configuration at Catalog > Inventory > Stock Options).
 
     - Set **Stock Alert Email Template** to the template that you want to use for product stock alerts.
 
     - Set **Alert Email Sender** to the [store contact]({% link stores/store-email-addresses.md %}) that you want to appear as the sender of the email alert.
 
+      |General Contact|The email address associated with the General Contact identity. |
```

Comment:
I think we should delete this list as the link I suggested above explains these options in more detail.

Reply:
@erikmarr 
Hi, thank you! Done✅ 
Please, approve.